README.md

```diff
@@ -16,14 +16,13 @@ npm install @eslint/eslintrc --save-dev
 yarn add @eslint/eslintrc -D
 ```
 
-## Future Usage
-
-**Note:** This package is not intended for public use at this time. The following is an example of how it will be used in the future.
+## Usage
 
 The primary class in this package is `FlatCompat`, which is a utility to translate ESLintRC-style configs into flat configs. Here's how you use it inside of your `eslint.config.js` file:
 
 ```js
-import { FlatCompat } from "@eslint/eslintrc";
+import pkg from "@eslint/eslintrc";
+const { FlatCompat } = pkg;
```

Comment:
looks like the original code example is correct. it was using named exports:https://github.com/eslint/eslintrc/blob/99adbac4fec25a50e466946833677351eb8794b9/lib/index.js#L50-L56

Reply:
We had a report that regular names imports weren’t working so we changed other examples to this format for best compatibility.

Reply:
no, it's just another case. eslint is a commonjs package, so users have do so, but this one does have an esm exports. It won't work to import like this.

Reply:
Ah right, thanks. I’ll fix this back to the original.